script/setup-kubeapps-harbor.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Constants
+ROOT_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )/.." >/dev/null && pwd)"
+
+# Load Libraries
+# shellcheck disable=SC1090
+. "${ROOT_DIR}/script/libtest.sh"
+# shellcheck disable=SC1090
+. "${ROOT_DIR}/script/liblog.sh"
+
+info "Updating chart repositories..."
+silence helm repo update
+echo
+
+# Install Harbor
+info "Installing Harbor in namespace 'harbor'..."
+harbor_values="$(cat << 'EOF'
+service:
+  tls:
+    enabled: false
+EOF
+)"
+info "Using the values.yaml below:"
+printf '#####\n\n%s\n\n#####\n' "$harbor_values"
+silence kubectl create ns harbor
+silence helm install harbor \
+  --namespace harbor \
+  -f <(echo "$harbor_values") \
+  bitnami/harbor
+# Wait for Harbor components
+info "Waiting for Harbor components to be ready..."
+harbor_deployments=(
+  "harbor-chartmuseum"
+  "harbor-clair"
+  "harbor-core"
+  "harbor-jobservice"
+  "harbor-nginx"
+  "harbor-notary-server"
+  "harbor-notary-signer"
+  "harbor-portal"
+  "harbor-registry"
```

Comment:
When demoing kubeapps-harbor integration, I usually use:
```
--set persistence.enabled=false \
--set postgresql.persistence.enabled=false \
--set redis.master.persistence.enabled=false \
--set jobservice.replicas=0 \
--set registry.replicas=0 \
--set clair.replicas=0 \
--set notary.enabled=false
```
so that less resources are required locally (I often use a kind cluster). I'm not saying you should do the same here, but it might be worth considering, or at least making them optional?

Reply:
I enabled the possibility to disable Clair, Notary and Chartmuseum.
We can improve this in the future to disable other features (e.g. persistence).